src/main/java/uk/gov/hmcts/darts/notification/service/impl/NotificationServiceImpl.java

```diff
@@ -73,6 +74,8 @@ private Notification saveNotificationToDb(String eventId, String caseId, String
 
 
     @Override
+    @SchedulerLock(name = "NotificationService_sendNotificationToGovNotify",
+        lockAtLeastFor = "PT5M", lockAtMostFor = "PT14M")
```

Comment:
I don't think we want to hold the lock for at least 5 minutes, maybe just 1 minute? The task runs every 2 minutes.

Also, not sure we want to hold the lock for 14 minutes? Maybe 5?